api/services/ocean/spark/schemas/oceanSparkApplicationState.yaml

```diff
@@ -0,0 +1,10 @@
+description: The current state of the Spark application
```

Comment:
Can we add additional description? I think we should explain the distinction between non-terminal states (new -> submitted -> running), and we should also explain the difference between Failed and Submission_failed.

Reply:
Thanks! Can you check the language?

Reply:
@jystephan for the notification